polyfill/RTCDataChannel.js

```diff
@@ -10,13 +10,12 @@ export default class _RTCDataChannel extends EventTarget {
     #negotiated;
     #ordered;
 
-    // needs to be called with .bind(this)
-    onbufferedamountlow = createEmptyFunction();
-    onclose = createEmptyFunction();
-    onclosing = createEmptyFunction();
-    onerror = createEmptyFunction();
-    onmessage = createEmptyFunction();
-    onopen = createEmptyFunction();
+    onbufferedamountlow;
```

Comment:
If we are calling like ` this.XXX?.(e);` then this is better

Reply:
yes, according to spec, these can be null, so we shouldnt initiate the